e2e-polybft/bridge_test.go

```diff
@@ -56,7 +56,10 @@ func checkLogs(
 }
 
 func TestE2E_Bridge_MainWorkflow(t *testing.T) {
-	const num = 10
+	const (
+		num                = 10
+		blockFinalityDepth = 4
```

Comment:
We should update the `cluster` script, to have a similar `blockFinalityDepth` since we also set the `epochSize` to 10 in it. Since the script is used for testing/debugging, I think it should have a `blockFinalityDepth` similar to Bridge e2e test.

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1228/commits/8c2218bb0b63abbc49def4f4ab58fd3be471c300
Set `--block-finality-depth` to 2